tests/test_server.py

```diff
@@ -161,6 +161,26 @@ def test_delete_prod_info(self):
         self.assertEqual(status.HTTP_200_OK, response.status_code)
         self.assertEqual(entry_count - 1, self.get_entry_count())
 
+    def test_update_prof_info(self):
+
+        #Test updating existing prod_id
+        test_prod_id = 1
+        prod_info = ProductInformation.find(test_prod_id)
```

Comment:
Why finding test_prod_id? `prod_id: 1` is already in database during setup.
We should trust setUp() doing the right thing.